launch/view_ur.launch.py

```diff
@@ -131,13 +136,24 @@ def generate_launch_description():
     )
     robot_description = {"robot_description": robot_description_content}
 
+    initial_positions_file = os.path.join(
+        get_package_share_directory("ur_description"), "config", "initial_positions.yaml"
+    )
+    print(initial_positions_file)
+    with open(initial_positions_file, "r") as stream:
+        try:
+            initial_positions = yaml.safe_load(stream)
```

Comment:
I don't like much this direct dependency on yaml.

Can we do something like in [Driver#308](https://github.com/UniversalRobots/Universal_Robots_ROS2_Driver/pull/308)